### ISSUE TITLE
Ignored b/c invalid config

### ISSUE THREAD

**Issue by Chrisissorry** [adammcarth/react-native-segmented-picker#28]
```
yarn run v1.22.5
$ react-native run-ios
warn Package react-native-segmented-picker has been ignored because it contains invalid configuration. Reason: Option dependency.platforms.ios must be a object, instead got object
```

Using RN 0.63.2

**Comment by adammcarth** [comment]
Thanks for the report @Chrisissorry :)

Oh yeah, I'm aware of this one. It's a strange error message that I _believe_ first popped up after installing `react-native-swift`. Are you using the `native` addon for iOS?

You can safely ignore the warning message for now, but I'll do my best to figure out how to fix it ASAP - because it's definitely annoying seeing it every time you do `yarn install` etc.

**Comment by Chrisissorry** [comment]
Yes I am using the native addon, but I saw this warning also when using the non-native version.

Actually I opened this issue because it would not render, but that could've been an issue on my side. I am pretty new to RN 😌

**Comment by erodrig** [comment]
It seems this is failng given the config file has null on it 


/**
 * Auto linking for iOS native code is disabled because it is an opt-in feature.
 * See manual installation instructions:
 */

module.exports = {
  dependency: {
    platforms: {
      android: null,
      ios: null,
    },
  },
};

**Comment by adammcarth** [comment]
Oh, good find @erodrig! I always assumed that error was being generated because of an invalid Podfile, but you're right - it looks like the `react-native.config.js` file is the culprit here.

I'm really not sure what to do though. The official React Native CLI documentation [explicitly says to set it as `null`](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#how-can-i-disable-autolinking-for-unsupported-library) to disable Auto Linking?

So I'm actually thinking this is a bug with the `react-native-swift` package that we might need to raise with them. `null` is supposed to be a valid value, so they shouldn't warn us against using it.

**Comment by adammcarth** [comment]
^ Whoops. Turns out it's actually an issue with the React Native CLI. I've raised a PR.
